Cargo.toml

```diff
@@ -5,11 +5,15 @@ resolver = "2"
 [workspace.dependencies]
 aws-lc-rs = "1.6.0"
 pem = "3.0.2"
-pki-types = { package = "rustls-pki-types", version = "1.3.0" }
+pki-types = { package = "rustls-pki-types", version = "1.4.0" }
```

Comment:
If you bump this to [1.4.1](https://github.com/rustls/pki-types/releases/tag/v%2F1.4.1) we can drop the patch below.